plugins-bundled/internal/input-datasource/webpack.config.ts

```diff
@@ -0,0 +1,146 @@
+import CopyWebpackPlugin from 'copy-webpack-plugin';
```

Comment:
This is a modified version from create-plugin. Simplified for this specific simple case.